File: app/src/main/java/in/woloo/www/trendingblog/mvp/NearByWolooAndOfferCountView.java
package in.woloo.www.trendingblog.mvp;

import in.woloo.www.more.models.UserProfileMergedResponse;
import in.woloo.www.trendingblog.model.CategoriesResponse;
import in.woloo.www.trendingblog.model.NearByWolooAndOfferCountResponse;
import in.woloo.www.trendingblog.model.blog.BlogsResponse;

public interface NearByWolooAndOfferCountView {
    void nearByWolooAndOfferCountResponse(NearByWolooAndOfferCountResponse nearByWolooAndOfferCountResponse);

    void getCategories(CategoriesResponse categoriesResponse);

    void getBlogs(BlogsResponse blogsResponse);

    void onFavouriteABlog();

    void onLikeABlog();

    void onReadABlog();

    void onBlogReadPointsAdded();

    void setUserProfileMergedResponse(UserProfileMergedResponse userProfileMergedResponse);
}

File: app/src/main/java/in/woloo/www/more/trendingblog/mvp/NearByWolooAndOfferCountView.kt
package `in`.woloo.www.more.trendingblog.mvp

import `in`.woloo.www.more.models.UserProfileMergedResponse
import `in`.woloo.www.more.trendingblog.model.CategoriesResponse
import `in`.woloo.www.more.trendingblog.model.NearByWolooAndOfferCountResponse
import `in`.woloo.www.more.trendingblog.model.blog.BlogsResponse

interface NearByWolooAndOfferCountView {
    fun nearByWolooAndOfferCountResponse(nearByWolooAndOfferCountResponse: NearByWolooAndOfferCountResponse?)
    fun getCategories(categoriesResponse: CategoriesResponse?)
    fun getBlogs(blogsResponse: BlogsResponse?)
    fun onFavouriteABlog()
    fun onLikeABlog()
    fun onReadABlog()
    fun onBlogReadPointsAdded()
    fun setUserProfileMergedResponse(userProfileMergedResponse: UserProfileMergedResponse?)
}
